### DISCUSSION TITLE
controls for objects in group without ungrouping + parent objects

### DISCUSSION THREAD

**Discussion by ebigtm** [fabricjs/fabric.js#7411] [Q&A]
Hi, after very long unsuccessful attemps to try and do it ourselves we would like to check with the posibilites of two things:

1. We have to give people the possiblity to **click on one or more groups at a time to edit their inside elements**, when finishing, the groups should be "closed" again, for now we are ungrouping and regrouping again afterwards but it has alot of limitations when handling more than one group at a time, we were wondering if theres an easy way to edit objects inside groups without ungroup (with controls for each) or even if we can extend the group class to do so, we just didnt figured out how ourselves.
2. We wondered if theres a possiblity to **"link" objects to eachother as parents/childs,** meaning to have the possiblity to control the child alone or if the parent selected it will move all its children together.

Thanks in advance :)

**Comment by asturur** [comment]
I also tried to make the situation better, with not much success.
I m not sure if you were on the other thread where we were discussing this.

**Reply by ebigtm** [reply]
No I wasn't there can you put a link? So. I guess there isn't a straight forward solution for that?

**Comment by ShaMan123** [comment]
#6776

**Reply by ebigtm** [reply]
Thank you! as mentioned there would really love to speek to you @ShaMan123 whenever you have time!
